components/button/index.css

```
@@ -121,11 +123,17 @@ governing permissions and limitations under the License.
   }
 
   .spectrum-Icon {
+    margin-block-start: max(0px,
+      var(--mod-button-top-to-icon, var(--spectrum-button-top-to-icon)) - 
+      var(--mod-button-border-width, var(--spectrum-button-border-width))
+    );
```

```suggestion
    margin-block-start: max(0px,
      var(--mod-button-top-to-icon, calc(var(--spectrum-button-top-to-icon)) - 
      var(--mod-button-border-width, var(--spectrum-button-border-width)))
    );
```

This part doesn't need to be wrapped in a calc?